helpers/validate.js

```diff
@@ -45,7 +45,7 @@ export default class ValidateUser {
   static validatePassword(str) {
     const passwordFormat = /((^[0-9]+[a-z]+)|(^[a-z]+[0-9]+))+[0-9a-z]+$/i;
     if (!str || str.trim().length < 1) {
-      return { error: 'Password cannnot be empty' };
+      return { error: 'Password cannot be empty' };
```

Comment:
Implement returning 'error' as stated in the readme.

Reply:
I believe this is a function I wrote. This return is from a helper method in this class and not from the json response. The Json response using this method returns errors in the right format.